File: Conditionals/TimeOfDay.py
#Eve Richardson Time of Day Python

import time

current = time.time()
now = time.ctime(current)
local_time = time.localtime(current)
hours = local_time.tm_hour

if hours <= 11:
    print("Good morning!\n")
elif hours <= 15:
    print("Good afternoon!\n")
else:
    print("Good evening!\n")

File: Conditionals/TimeOfDay.c
// Eve Richardson, Time of day C
#include <stdio.h>
#include <time.h>

int main(void){
    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;

    if (hour <= 11){
        printf("Good morning!\n");
    }else if (hour <=15){
        printf("Good afternoon!\n");
    }else{
        printf("Good evening!\n");
    }

    return 0;
}